wear/src/main/java/io/homeassistant/companion/android/sensors/HealthServicesSensorManager.kt

```diff
@@ -31,13 +41,26 @@ class HealthServicesSensorManager : SensorManager {
             entityCategory = SensorManager.ENTITY_CATEGORY_DIAGNOSTIC,
             updateType = SensorManager.BasicSensor.UpdateType.INTENT
         )
+        private val dailyFloors = SensorManager.BasicSensor(
+            "daily_floors",
+            "sensor",
+            commonR.string.sensor_name_daily_floors,
+            commonR.string.sensor_description_daily_floors,
+            "mdi:stairs",
+            unitOfMeasurement = "floors",

```

Comment:
Shouldn't this string be localizable? It isn't really an unit.

Reply:
We don't localize `unitOfMeasurement` anywhere though so would be strange to do it in one place.

Reply:
Are there strings like this used anywhere else? It is not a standard unit so if the install is not in English "floors" will look out of place.

Reply:
yes steps and also bluetooth connection, I believe if `mobile_app` integration provides the strings in core it should pick up the translations from there just like some state updates

Reply:
Testing with the Bluetooth connection sensor and I'm not seeing a translation - the state shows up as "1 connection(s)" even when HA is not in English. 

Not a problem for this PR to solve so considering that is already used elsewhere doing it like this is fine for now 👍